defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java

```diff
@@ -75,6 +75,10 @@
 
 	@DeleteMapping(value = EndpointURI.DELETE_EMPLOYEE)
 	public ResponseEntity<Object> deleteEmployee(@PathVariable Long id) {
+		if (!employeeService.isEmployeeAlreadyExists(id)) {
+			return new ResponseEntity<>(new ValidationFailureResponse(ValidationConstance.EMPLOYEE_NOT_EXISTS,
+					validationFailureStatusCodes.getEmployeeAlreadyExist()),HttpStatus.BAD_REQUEST);
```

Comment:
how can you use getEmployeeAlreayExist() method? this is for not exist?

Reply:
yah you used not equal but when the id not exists it will return the status code of the getEmployeeAlreadyExist()..